buildSrc/build.gradle.kts

```diff
@@ -1,10 +1,11 @@
 plugins {
-    kotlin("jvm") version "1.7.22"
+    `kotlin-dsl`
 }
 
 kotlin {
     jvmToolchain {
-        (this as JavaToolchainSpec).languageVersion.set(JavaLanguageVersion.of(8))
+        languageVersion.set(JavaLanguageVersion.of(8))
+        vendor.set(JvmVendorSpec.AZUL)
```

Comment:
just curious, why did we choose Azul? is it because they provide an m1 compatible java 8 jdk?

Reply:
Yes, that's correct.